### DISCUSSION TITLE
Changing left style of a multiselect

### DISCUSSION THREAD

**Discussion by soundstep** [charmbracelet/lipgloss#369] [Q&A] [answered]
Hi,

Beginner question, I'm using that [MultiSelect](https://github.com/charmbracelet/huh#multiple-select).

How to remove that first `|`, so that this:

```sh
┃ > • 30 Nov 2021 | Text`
```

Becomes this:

```sh
> • 30 Nov 2021 | Text`
```

I'm trying to create a custom theme like [this](https://github.com/charmbracelet/huh/blob/v0.6.0/theme.go#L111-L153), but I can't find the part that sets this.

**Comment by soundstep** [comment, marked as answer]
Fund it I think:

```sh
t := huh.ThemeBase()
t.Focused.Base = lipgloss.NewStyle().PaddingLeft(0).BorderStyle(lipgloss.HiddenBorder()).BorderLeft(false)
```

**Reply by bashbunni** [reply]
Hey @soundstep did you figure this out? The solution you posted looks about right

**Reply by soundstep** [reply]
Yes, it seems to work, thanks @bashbunni. It was not too easy to find, but it might be my go shortcomings!

**Reply by bashbunni** [reply]
I can imagine it was tricky to find! It's not really documented anywhere, but hopefully this solution can help others! Thank you for sharing it 🙏
